src/manage_nsfs/manage_nsfs_cli_utils.js

```diff
@@ -55,6 +55,22 @@ async function get_config_data(config_root_backend, config_file_path, show_secre
     return config_data;
 }
 
+/**
+ * silent_get_config_data will read a config file and return its content 
+ * while omitting secrets if show_secrets flag was not provided
+ * if the config file was deleted (encounter ENOENT) - continue
+ * @param {string} config_file_path
+ * @param {boolean} [show_secrets]
+ */
+async function silent_get_config_data(config_root_backend, config_file_path, show_secrets = false) {
+    try {
+        const config_data = await get_config_data(config_root_backend, config_file_path, show_secrets);
+        return config_data;
+    } catch (err) {
+        if (err.code !== 'ENOENT') throw err;
```

Comment:
@shirady Can you add dbg.warn() error message?

Reply:
@shirady Not on this PR, we need to rethink about the list error handling...
Also, we wanted to not fail on master keys manager failures as well, See - https://github.com/noobaa/noobaa-core/issues/8104
We should put the try & catch on the loop content, and return the entry name of the listed entity with the error...

Reply:
@romayalon I suggest opening an issue or managing it with a task, so we will not forget.

Reply:
@shirady we already have a bug - https://github.com/noobaa/noobaa-core/issues/8104

Reply:
The original issue is about accounts, your comment here expands it to buckets (in noobaa CLI and S3 API).

Reply:
Ok, so we can expand the original issue, why don't need a different bug for buckets

Reply:
Sure